src/consts/readOnly.ts

```diff
@@ -0,0 +1,228 @@
+      currentIndex = currentIndex + step
+    ) {
+      const newRemovalsList = fromResultToList(fullRemovalsList, currentIndex, currentIndex + step);
+      const newResult = await ReadOnlyFunctions([newRemovalsList], 'get-all-data-miners-proposed-for-removal');
+
+      if (newResult) {
+        newResultList.push(cvToJSON(newResult));
+      }
+    }
+    return newResultList;
+  };
+
+// get-all-data-miners-pending-accept (pending-miners-list (list 100 principal))
+// args: 
+// what does it do:
+// return:
+//
+// get-remaining-blocks-until-join
+// args: none
+// what does it do: Gets the number of blocks left until a miner can accept the users that are in pending list
+// return: Remaining blocks, number
+
+export const readOnlyGetRemainingBlocksJoin = async () => {
+    const blocksLeft = await ReadOnlyFunctions([], 'get-remaining-blocks-until-join');
+    return Number(convertCVToValue(blocksLeft));
+  };
+
+// get-all-data-miners-blocks (local-miners-list (list 100 principal))
+// args: 
+// what does it do: get how many blocks passed since miner in list joined
+// return:
+//
+// get-data-notifier-election-process
+// args: 
+// what does it do: returns notifier voting status and the blocks remaining until the end (if > blockheight)
+// return:
+//
+// get-all-data-notifier-voter-miners (voter-miners-list (list 100 principal))
+// args: 
+// what does it do: returns the miner and which notifier it voted for each miner
+// return:
+//
+// was-block-claimed (given-block-height uint)
+// args: 
+// what does it do: true/false if rewards on the block were claimed
+// return:
+//
+// get-all-data-balance-miners (local-miners-list (list 100 principal))
+// args: 
+// what does it do: returns the stx balance for every miner
+// return:
+//
+// get-balance (address principal)
+// args: 
+// what does it do: returns balance for given address
+// return:
+//
+// get-principals-list (address principal)
+// args: 
+// what does it do:
+// return:
+//
+// get-k
+// args: 
+// what does it do: threshold for notifier votes
+// return:
+//
+// get-notifier
+// args: 
+// what does it do:
+// return:
+//
+// get-waiting-list
+// args: none
+// what does it do: returns a list of miners that are in waiting list
+// return: waiting miners list
+
+export const ReadOnlyGetWaitingList = async () => {
+    const waitingList: ClarityValue = await ReadOnlyFunctions([], 'get-waiting-list');
+    return waitingList;
+  };
+
+// get-miners-list
+// args: none
+// what does it do: returns a list of miners that are in pool
+// return: miners in pool list
+
+export const ReadOnlyGetMinersList = async () => {
+    const minersList = cvToJSON(await ReadOnlyFunctions([], 'get-miners-list'));
+    return minersList;
+  };
+
+// get-pending-accept-list
+// args: 
+// what does it do:
+// return:
+//
+// get-notifier-vote-number (voted-notifier principal)
+// args: 
+// what does it do: get the votes for a given notifier
+// return:
+//
+// get-max-voted-notifier
```

Comment:
Are this comments stuff you'll add later?

Reply:
Yes, I will add them all tomorrow, and further change all 'any' types I find with the correct ones. For now, I'll commit with these changes.

Reply:
I just added comments for what each of the functions in "readOnly.ts" and "smartContractFunctions.ts" files do, take as arguments, or return. @amalexandrescu can add them when they are needed, by looking at the ones already integrated.